aws/terminator/data_services.py

```diff
@@ -199,6 +199,48 @@ def terminate(self):
         self.client.delete_db_instance(DBInstanceIdentifier=self.name, SkipFinalSnapshot=True)
 
 
+class RdsDbSnapshot(Terminator):
+    @staticmethod
+    def create(credentials):
+        return Terminator._create(credentials, RdsDbSnapshot, 'rds', lambda client: client.describe_db_snapshots()['DBSnapshot'])
```

Comment:
```suggestion
        return Terminator._create(credentials, RdsDbSnapshot, 'rds', lambda client: client.describe_db_snapshots()['DBSnapshots'])
```
I'm getting a missing key error here. Should this be `DBSnapshots`?